code/modules/mob/living/simple_mob/subtypes/mechanical/disbot_vr.dm

```diff
@@ -0,0 +1,50 @@
+/mob/living/simple_mob/mechanical/infectionbot
+	name = "Strange robot"
+	desc = "You get the feeling you should run."
+	icon = 'icons/mob/vagrant_vr.dmi'
+	icon_state = "vagrant"
+	icon_living = "vagrant"
+	icon_dead = "vagrant"
+	icon_gib = "vagrant"
+
+	maxHealth = 65
+	health = 40
+	movement_cooldown = 4
+
+	response_help = "pets the"
+	response_disarm = "gently pushes aside the"
+	response_harm = "hits the"
+	faction = "vagrant"
+	harm_intent_damage = 3
+	melee_damage_lower = 6
+	melee_damage_upper = 9
+	light_color = "#8a0707"
+	attacktext = "mauled"
+	attack_sound = 'sound/weapons/bite.ogg'
+
+	ai_holder_type = /datum/ai_holder/simple_mob/melee
+	say_list_type = /datum/say_list/disbot
+
+	var/poison_chance = 100
+	var/poison_per_bite = 5
+	var/poison_type = "zombiepowder"

```

Comment:
Zombie Powder is awful choice for an attack poison... Considering that it puts person out of commision for the longest time just for one hit. Nothing is less fun than having to wait several minutes after every encounter with one like that just for effects to wear off. And thats not counting that it will rapid-inject it more and more over time even after target is long down. It does not discourage facetanking, it encourages overprepping on carbon.

Reply:
Carbon won't do anything. There actually is no way to 'counter' this other than not getting jabbed in the first place. Regardless of the chems you stock up on beforehand, if you get jabbed, you're muted and stunned for 60 seconds for each time it hits you. If it doesn't stop hitting you, you're just kinda screwed and you're going to die.